exercises/concept/treasure-chest/.meta/design.md

```diff
@@ -0,0 +1,26 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach students how to consume and make generic code.
+
+## Learning objectives
+
+- Understand what a type variable is.
+- Know how to define generic types.
+- Know how to define generic functions.
+
+## Out of scope
+
+- Covariance and contravariance.
+- Higher order functions.
```

Comment:
I think we have the bandwidth for talking about higher order functions, we can add one paragraph with `List.map` like I suggested before.

Reply:
I've discussed this in answer to another comment ...